content/influxdb/v2.4/get-started/visualize.md

```diff
@@ -11,21 +11,174 @@ menu:
+        {{< /expand-wrapper >}}
+
+    2.  Use the data picker to select the stop and stop date and time or manually
+        enter the following start and stop times:
+
+        - **Start**: 2022-01-01 08:00:00
+        - **Stop**: 2022-01-01 20:00:01
+    
+    3. Click **{{% caps %}}Apply Time Range{{% /caps %}}**.
+
+5.  Use the **Query Builder** to select the measurement, fields, and tags to query:
+
+    1. In the **{{% caps %}}From{{% /caps %}}** column, select the **get-started** bucket.
+    2. In the **Filter** column, select the **home** measurement.
+    3. In the next **Filter** column, select the **temp** field.
+    4. In the next **Filter** column, select the **room** tag and the **Kitchen** tag value.
+
+6.  Click **{{% caps %}}Submit{{% /caps %}}** to run the query and visualize the
+    results.
+
+    {{< img-hd src="/img/influxdb/2-4-get-started-visualize-query-builder.png" alt="InfluxDB Query Builder" />}}
+
+7. Click **{{< icon "check" >}}** to save the cell and return to the dashboard.
+
+## Create and use dashboard variables
+
+InfluxDB dashboard cells use **dashboard variables** to dynamically change
+specific parts of cell queries.
+The query builder automatically builds queries using the following
+[predefined dashboard variables](/influxdb/v2.4/visualize-data/variables/#predefined-dashboard-variables),
+each controlled by selections in your dashboard:
+
+- `v.timeRangeStart`: Start time of the queried time range specified by the time range selector.
+- `v.timeRangeStop`: Stop time of the queried time range specified by the time range selector.
+- `v.windowPeriod`: Window period used downsample data to one point per pixel in
+  a cell visualization. The value of this variable is determined by the pixel-width of the cell.
+
+### Create a custom dashboard variable
+
+Let's create a custom dashboard variable that we can use to change the field 
+displayed by your dashboard cell.
+
+1.  Select **Settings > Variables** in the left navigation bar.
+
+    {{< nav-icon "settings" >}}
+
+2.  Click **+ {{% caps %}}Create Variable{{% /caps %}}** and select **New Variable**.
+3.  Name your variable. For this tutorial, name the variable, **room**.
+4.  Select the default **Query** dashboard variable type.
+    This variable type uses the results of a query to populate the list of potential
+    variable values. _For information about the other dashboard variable types,
+    see [Variable types](/influxdb/v2.4/visualize-data/variables/variable-types/)._
+5.  Enter the following Flux query to return all the different `room` tag values
+    in your `get-started` bucket from the Unix epoch.
+
+    ```js
+    import "influxdata/influxdb/schema"
+
+    schema.tagValues(bucket: "get-started", tag: "room", start: time(v: 0))
+    ```
+
+6. Click **{{% caps %}}Create Variable{{% /caps %}}**.
+
+### Use a custom dashboard variable
+
+1.  Navigate to your **Getting Started Dashboard**.
+
+    {{< nav-icon "dashboards" >}}
```

Comment:
the icon here is the Data Explorer icon, the dashboard icon is different in both OSS 2.5 and c2